scripts/redcap/wrong_date_associations.py

```diff
@@ -148,7 +150,41 @@ def log_row(row: pd.Series, **kwargs):
     for _, row in errors_df.reset_index().iterrows():
         log_row(row, uid=uid_template, **kwargs)
 
+def get_special_cases(marks, session):
+    """
+    Looks at special_cases.yml file, finds the special cases under the file name section,
+    and then excludes any cases that match the special cases list.
+
+    Note (for Chris use):
+    - 'redcap_event_name' is arm name
+    - 'study_id' inclues up to letter and number
+    - 'visit_date' is looking at date --> not correct!
 
+    Each row in the dataframe has:
+    row.name -> [0] is the subject ID, [1] is the arm name
```

Comment:
Note that that's only true if you're iterating through the `DataFrame.iterrows()` and `row` is a `pd.Series`.

Reply:
Just added a small comment to the docs. Again, more of a personal note, but I also know -- and appreciate greatly -- that you like extensive + concise documentation.